### PR TITLE
revise content of requesting access page

### PR DESCRIPTION
**Pull request by asjohnston-asf**

### CODE REVIEW

File: docs/using/requesting_access.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+You must request access before submitting jobs for On Demand processing using the HyP3 platform. HyP3 accounts are limited to one per person.
+
+[Request Access](#){ .md-button .md-button--primary .middle }
+
+You will need an [Earthdata Login](https://urs.earthdata.nasa.gov/ 'https://urs.earthdata.nasa.gov/' ){target=_blank} (EDL) account to complete this request. You can [register here](https://urs.earthdata.nasa.gov/users/new 'https://urs.earthdata.nasa.gov/users/new' ){target=_blank} if you do not already have an account. 
+
+You will receive an email with your approval status within two business days. If you have not already registered for HyP3 access, you can expect to be approved.
+
+Once your access request has been approved, you can submit jobs for On Demand processing using your EDL credentials. *You only need to complete this approval process once.* 
+
```
Comment (hjkristenson) [inline docs/using/requesting_access.md:12]: I think there still is merit to including some clarification on the grandfathering process. Maybe moving it down to this location? 
```suggestion

!!! tip "Existing HyP3 users do not need to request access" 

    Existing On Demand users will *not* need to undergo this process unless they have been identified as having multiple HyP3 accounts. If you are an existing user who is no longer able to process On Demand jobs, you can either submit an [access request form](#TODO: add link to request form) or email uso@asf.alaska.edu for more information.
```

**Review by hjkristenson — APPROVED** [review]
This looks OK as-is, though I would prefer to include language clarifying that existing users don't need to apply for access. This also would cover those who had multiple accounts and try to submit something with one of those accounts after they've lost access. I've made a suggestion, but I'm open to being overruled if there are concerns about adding that back in.

File: docs/using/requesting_access.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+!!! tip "Existing HyP3 users do not need to request access" 
+
+    Existing On Demand users will *not* need to undergo this process unless they have been identified as having multiple HyP3 accounts. If you are an existing user who is no longer able to process On Demand jobs, you can either submit an [access request form](#TODO: add link to request form) or email uso@asf.alaska.edu for more information.
```
Comment (forrestfwilliams) [inline docs/using/requesting_access.md:15]:
```suggestion
    Existing On Demand users (i.e., users who signed up before April 2024) will *not* need to undergo this process. If you are no longer able to process On Demand jobs and you believe you should still have access, you can either submit an [access request form](#TODO: add link to request form) or email uso@asf.alaska.edu for more information.
```

File: docs/using/requesting_access.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+You will need an [Earthdata Login](https://urs.earthdata.nasa.gov/ 'https://urs.earthdata.nasa.gov/' ){target=_blank} (EDL) account to complete this request. You can [register here](https://urs.earthdata.nasa.gov/users/new 'https://urs.earthdata.nasa.gov/users/new' ){target=_blank} if you do not already have an account. 
+
+You will receive an email with your approval status within two business days. If you have not already registered for HyP3 access, you can expect to be approved.
+
+Once your access request has been approved, you can submit jobs for On Demand processing using your EDL credentials. *You only need to complete this approval process once.* 
+
+!!! tip "Existing HyP3 users do not need to request access" 
+
+    Existing On Demand users (i.e., users who signed up before April 2024) will *not* need to undergo this process. If you are no longer able to process On Demand jobs and you believe you should still have access, you can either submit an [access request form](#TODO: add link to request form) or email uso@asf.alaska.edu for more information.
```
Comment (forrestfwilliams) [inline docs/using/requesting_access.md:15]:
```suggestion
    Existing On Demand users (i.e. users who signed up before May 2024) will *not* need to undergo this process. If you are no longer able to process On Demand jobs and you believe you should still have access, you can either submit an [access request form](#TODO: add link to request form) or email uso@asf.alaska.edu for more information.
```

**Review by forrestfwilliams — APPROVED** [review]

**Pull request merged**